src/Components/Common.js

```diff
@@ -14,16 +14,28 @@ export const gqlProps = {
 
 export const  isBeta = () => window.location.pathname.split('/')[1] === 'beta' ? '/beta' : '';
 
+/* eslint-disable max-len, padding-line-between-statements */
 export const expandMatchMetadata = (md) => {
-    let output = '';
-    md.source_type === 'file' ? (
-        output += 'File Type: ' + md.file_type,
-        output += '\nFile Mime Type: ' + md.mime_type,
-        output += '\nFile MD5Sum: ' + md.md5sum,
-        'line' in md ? (
-            output += '\nMatching Line Number: ' + md.line_number,
-            output += '\nMatching Line: ' + decodeURIComponent(md.line)
-        ) : null)
-        : output += 'Process Name: ' + md.process_name;
+    let output = 'Source type: ' + md.source_type;
+    if (md.source_type === 'file') {
+        // scanned source was a file, parse its metadata (if any)
+        if ('file_type' in md) {
```

Comment:
great candidate for a template literal https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals (totally encourage the read)

TL;DR 

``File Type: ${md.file_type}
File Mime type: ${md.mime_type}
File MD5Sum: ${md.md5sum}
${md.line ? `Matching Line Number: ${md.line_number}
Matching Line: ${decodeURIComponent(md.line}
... ... 
``

Reply:
Well I managed to do it.  Code seems a bit unreadable to me now, but at least it uses template literals and a smattering of tertiary conditionals.